ui/src/app/app-routing.module.ts

```diff
@@ -346,6 +341,10 @@ const routes: Routes = [
         data: {title: 'Create Workspace', mode: WorkspaceEditMode.Create}
       }
     ]
+  }, {
+    path: '**',
+    component: AppRouting,
+    data: {title: 'Sign In'}
```

Comment:
why "Sign In" ?

Reply:
I left this in accidentally - removed - thanks!